content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -193,6 +232,50 @@ it does the following when a Pod is created:
 1. If the spec of the incoming Pod doesn't already contain any `imagePullSecrets`, then the
    admission controller adds `imagePullSecrets`, copying them from the `ServiceAccount`.
 
+### Legacy ServiceAccount token tracking controller
+
+{{< feature-state for_k8s_version="v1.28" state="stable" >}}
+
+This controller generates a ConfigMap called
+`kube-system/kube-apiserver-legacy-service-account-token-tracking` in the
+`kube-system` namespace. The ConfigMap records the timestamp when legacy service
+account tokens began to be monitored by the system.
+
+### Legacy ServiceAccount token cleaner
+
+{{< feature-state for_k8s_version="v1.29" state="beta" >}}
+
+The legacy ServiceAccount token cleaner runs as part of the
+`kube-controller-manager` and checks every 24 hours to see if any auto-generated
+legacy ServiceAccount token has not been used in a *specified amount of time*.
+If so, the cleaner marks those tokens as invalid.
+
+The cleaner works by first checking the ConfigMap created by the control plane
+(provided that `LegacyServiceAccountTokenTracking` is enabled). If the current
+time is a *specified amount of time* after the date in the ConfigMap, the
+cleaner then loops through the list of Secrets in the cluster and evaluates each
+Secret that has the type `kubernetes.io/service-account-token`.
```

Comment:
These are implementation details, right?
As a user or an operator, skipping this paragraph doesn't hurt, right?

Reply:
In other words, is there a reason for a user to care about the ConfigMap thing?

Reply:
Yeah. The cleaner initially inspects the tracking date within the ConfigMap to assess if the tracking period is sufficient. Without understanding the role of the ConfigMap, users might be puzzled as to why certain secrets that are seemingly old enough are being invalidated right away.